Donuts/Donut.java

```diff
@@ -0,0 +1,65 @@
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+class Donut {
+	static double A = 1;
+	static double B = 1;
+
+	public static void main(String[] args) throws InterruptedException, IOException {
+		final boolean isWindows = System.getProperty("os.name").contains("Windows");
+		while(true) {
+			if (isWindows) {
+				new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor();
+			} else {
+				Runtime.getRuntime().exec("clear");
```

Comment:
Proposed change:
```new ProcessBuilder("clear").inheritIO().start().waitFor(); ```

Reply:
Should be fixed by d9710f4ef8547a98150441110dc30d04de1c84d5